tensorboard/webapp/notification_center/_data_source/notification_center_data_source.ts

```diff
@@ -32,4 +33,15 @@ export class TBNotificationCenterDataSource
   fetchNotifications(): Observable<NotificationCenterResponse> {
     return this.http.get<NotificationCenterResponse>(`data/notifications`);
   }
+
+  updateAndGetLastReadTimeStampInMs(): Observable<number> {
```

Comment:
I do not understand the contract here: what do you get by reading the previous `window.localStorage.getItem(NOTIFICATION_LAST_READ_TIME_KEY)` value?